release/gcb/local.sh

```diff
@@ -6,8 +6,16 @@ set -o errexit
 source gcb_lib.sh
 ROOT=$(cd "$(git rev-parse --show-cdup)" && pwd || return)
 artifacts="$HOME/output/local"
-export NEW_VERSION="fake"
-export DOCKER_HUB='docker.io/fake'
+export NEW_VERSION=${TAG:-}
+export DOCKER_HUB=${DOCKER_HUB:-}
+if [ -z "${NEW_VERSION}" ]; then
+   echo "Provide or export the tag for the images. TAG=my_amazing_tag."
+   exit 1
+fi
+if [ -z "${DOCKER_HUB}" ]; then
```

Comment:
Would it break things to make this just `HUB`? Just for some consistency since we use that in most places. If not its no problem, since it will fail and give the error message